src/helpers/permutateArray.helper.ts

```diff
@@ -0,0 +1,17 @@
+export const permutationsOfArray = (arr): string[] => {
```

Comment:
I am not sure how this works - let's discuss it during the next class session :)